# Patch written from Giovanni Santini <giovannisantini93@yahoo.it>
# Feel free to use it, just give proper credits writing in your patch:
# Original patch written from Giovanni Santini <giovannisantini93@yahoo.it>
# See: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=java-gnome
# Disable doclint, see: http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html
diff -ur a/java-gnome-4.1.3/build/faster b/java-gnome-4.1.3/build/faster
--- a/java-gnome-4.1.3/build/faster	2013-05-05 03:08:23.000000000 +0200
+++ b/java-gnome-4.1.3/build/faster	2015-07-13 11:33:55.207518811 +0200
@@ -650,6 +650,7 @@
 	cmd += "-d doc/api "
 	cmd += "-public "
 	cmd += "-nodeprecated "
+	cmd += "-Xdoclint:none "
 	cmd += "-source 1.5 "
 	cmd += "-notree "
 	cmd += "-noindex "
